packages/website/pages/login/index.js

```diff
@@ -91,6 +91,7 @@ const Login = () => {
             className={clsx('login-email', errors.email && 'error')}
             placeholder={pageContent.form_placeholder}
             onChange={useCallback(e => setFormData({ email: e.currentTarget.value }), [])}
+            onKeyDown={e => e.key === 'Enter' && onLoginWithEmail()}
```

Comment:
This should be in a `<form>` element with an `onSubmit` handler. Then the key press listener is not required.